styles/mixins/headings.css

```diff
@@ -0,0 +1,35 @@
+/*---
+section: Mixins
+title: Headings
+---
+
+```
+@mixin heading $size, $mobileSize;
+@mixin heading var(--heading-1-size), var(--heading-1-mobile-size);
+```
+Outputs
+
+```css
+.heading-1 {
+  font-family: serif;
+  color: #333;
+  line-height: 120%;
+  font-size: 45px;
+}
+@media (--mobile) {
+  .heading-1 {
+    font-size: 30px;
+  }
+}
+```
+*/
+
+@define-mixin heading $size, $mobileSize {
```

Comment:
another option I could do is have everything allowed to be passed in and set the global vars as the defaults.

`@define-mixin heading $size, $mobileSize, $family: var(--font-heading), $color: var(--color-heading)...)`